documentation/src/test/java/example/registration/WebServerDemo.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+package example.registration
+
+import org.junit.jupiter.api.Assertions.assertEquals
+
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.BeforeAllCallback
+import org.junit.jupiter.api.extension.ExtensionContext
+import org.junit.jupiter.api.extension.RegisterExtension
+
+// tag::user_guide[]
+class WebServerDemo {
+    // end::user_guide[]
+    // @formatter:off
+    // tag::user_guide[]
+    companion object {
+        @JvmField
+        @RegisterExtension
+        val server = WebServerExtension.builder()
+                .enableSecurity(false)
+                .build()
+    }
+    // end::user_guide[]
+    // @formatter:on
```

Comment:
I don't think we need the formatter comments here, do we?

Reply:
Yes, we do: for the builder pattern. Otherwise, that ends up being a single line.

It's of course a matter of taste, but this PR just copies the style I chose for the Java example. So the examples are consistent in that regard. 😉

Reply:
We do for Java, but for Kotlin we currently don't have a formatter that would join those lines. Thus, we should be able to remove them.